src/pykeen/losses.py

```diff
@@ -178,6 +180,40 @@ def __init__(self, size_average=None, reduce=None, reduction: str = 'mean'):
         super().__init__(size_average=size_average, reduce=reduce, reduction=reduction)
         self._reduction_method = _REDUCTION_METHODS[reduction]
 
+    def process_slcwa_scores(
+        self,
+        positive_scores: torch.FloatTensor,
+        negative_scores: torch.FloatTensor,
+        label_smoothing: Optional[float] = None,
+        batch_filter: Optional[torch.BoolTensor] = None,
```

Comment:
hmm looks like we'll want to wait on #447 to finish this one

Reply:
This was already present in `master`'s loss helpers.

Reply:
🦜